sdk/include/ddk/mountdev.h

```diff
@@ -1,8 +1,38 @@
+/*
+ * mountmgr.h
+ *
+ * Mount Manager mounted devices interface
+ *
+ * This file is part of the ReactOS DDK package.
+ *
+ * Contributors:
+ *   Magnus Olsen <greatlord@reactos.org>
+ *   Amine Khaldi <amine.khaldi@reactos.org>
+ *   Hermès Bélusca-Maïto <hermes.belusca-maito@reactos.org>
+ *
+ * THIS SOFTWARE IS NOT COPYRIGHTED
+ *
+ * This source code is offered for use in the public domain. You may
+ * use, modify or distribute it freely.
+ *
+ * This code is distributed in the hope that it will be useful but
+ * WITHOUT ANY WARRANTY. ALL WARRANTIES, EXPRESS OR IMPLIED ARE HEREBY
+ * DISCLAIMED. This includes but is not limited to warranties of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ */
+
 #ifndef _MOUNTDEV_
 #define _MOUNTDEV_
 
+#pragma once
```

Comment:
No I won't. Because the include-guards are also existing in the official headers, and they permit to know whether (for example) mountdev.h has been already included: let's suppose there is a header of a library that needs the definition of some mountdev structs or whatnot, it could do this:
```c
#ifndef _MOUNTDEV_ 
typedef struct _MOUNTDEV_NAME
{
...
} MOUNTDEV_NAME;
#endif
```
(bear in mind that I'm giving here a random example.)